bundles/org.openhab.binding.miele/README.md

```diff
@@ -502,3 +501,15 @@ sitemap miele label="Miele" {
     }
 }
 ```
+
+## Note on Discovery
```

Comment:
Include this in Discovery chapter (line 27) or include as subchapter?

Reply:
Ok. Done.